docs/source/configuration/multinode_cassandra.rst

```diff
@@ -0,0 +1,112 @@
+=================================
+Multinode Cassandra HA Guarantees
+=================================
+
+A multinode Cassandra cluster can generally be in one of three states:
+
+    1. All nodes are up.
+    #. Nodes down, but there is a quorum of up nodes (more than half the nodes are up).
+    #. Less than a quorum of the nodes are up.
+
+In the case where all nodes are up, the entire KVS API can be used. In the latter two cases, in order to guarantee consistency and correctness, usage of the KVS API is restricted as documented below.
+
+A quorum (but not all) of nodes are up
```

Comment:
Nit: In my head, this reads like "A quorum of nodes .... but not all of nodes". It would be better as "A quorum of nodes are up, but not all".